third_party/UglifyJS2/README.md

```diff
@@ -0,0 +1,17 @@
+# Contents
+Slim copy of [UglifyJS2](https://github.com/mishoo/UglifyJS2) output library.
+
+# Original Copy
+https://raw.githubusercontent.com/mishoo/UglifyJS2/v2.6.1/lib/output.js
+
+# Modifications
+
+- Removed all functions except `encode_string`.
+- Replaced call `var ret = make_string(str, quote)` with `var ret = str`
+- Added default `options` object
+- Added CommonJS `module.exports` line
```

Comment:
`module.exports` itself is an invention of Node.js. CommonJS Modules/1.1 spec says that only the plain `exports` variable is allowed to be the means of exporting things.